src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,10 @@
+$day-size: 100px;
+$calendar-padding: 10px;
+$calendar-gap: 1px;
+$start-day: "mon" 0,
+  "tue"$day-size +$calendar-gap,
+  "wed"($day-size * 2)+($calendar-gap * 2),
+  "thu"($day-size * 3)+($calendar-gap * 3),
+  "fri"($day-size * 4)+($calendar-gap * 4),
+  "sat"($day-size * 5)+($calendar-gap * 5),
+  "sun"($day-size * 6)+($calendar-gap * 6);
```

Comment:
looks like repeated logic, it's better to move those calculation into loop, where you set margin